src/view/com/pager/FeedsTabBar.web.tsx

```diff
@@ -58,7 +57,7 @@ function FeedsTabBarPublic() {
           style={[pal.text, {fontWeight: 'bold'}]}
           text={
             <>
-              {isSandbox ? 'SANDBOX' : 'Bluesky'}{' '}
+              Bluesky{' '}
               {/*hasNew && (
                 <View
                   style={{
```

Comment:
Can we remove this comment?

Reply:
Oh yeah looks like an artifact from the public web view sprint, can remove